pkg/config/environment_containers.go

```diff
@@ -157,6 +162,12 @@ func detectCloudFoundry(features FeatureMap) {
 	}
 }
 
+func detectPodman(features FeatureMap) {
+	if _, err := os.Stat(defaultPodmanContainersStoragePath); err == nil {
```

Comment:
You need to deal with host mount prefix, see below `getHostMountPrefixes`

Reply:
Right. I forgot about this. Should be OK now.